ChiTech/ChiPhysics/PhysicsMaterial/transportxsections/prop_transportxs_01b_readfromchi.cc

```diff
@@ -24,20 +24,161 @@ SIGMA_T_END
+  - NU_PROMPT_BEGIN. Optional. Starts a block that is terminated by a line
+    NU_PROMPT_END. Each line in the block processes the first two words as
+    [Group, nu]. Populates the nu_p_sigma_fg field. Upon completing the file
+    processing the nu_p_sigma_fg field gets multiplied by sigma_fg.
+    Symbol \f$ \nu_{prompt,g} \f$.
+  - NU_DELAYED_BEGIN. Optional. Starts a block that is terminated by a line
+    NU_DELAYED_END. Each line in the block processes the first two words as
+    [Group, nu]. Populates the nu_d_sigma_fg field. Upon completing the file
+    processing the nu_d_sigma_fg field gets multiplied by sigma_fg.
+    Symbol \f$ \nu_{delayed,g} \f$.
+  - CHI_PROMPT_BEGIN. Optional. Starts a block that is terminated by a line
+    CHI_PROMPT_END. Each line in the block processes the first two words as
+    [Group, chi]. Populates the chi_g field. Symbol \f$ \chi_{g} \f$.
+  - DDT_COEFF_BEGIN. Optional. Starts a block that is terminated by a line
+    DDT_COEFF_END. Each line in the block processes the first two words as
+    [Group, ddt_coeff]. Populates the ddt_coeff field.
+    Symbol \f$ \frac{1}{v_g} \f$.
+
+  - PRECURSOR_LAMBDA_BEGIN. Optional. Starts a block that is terminated by a line
+    PRECURSOR_LAMBDA_END. Each line in the block processes the first two words as
+    [precursor, lambda]. Populates the lambda field (the precursor decay
+    constant). Symbol \f$ \lambda_j \f$.
+  - PRECURSOR_GAMMA_BEGIN. Optional. Starts a block that is terminated by a line
+    PRECURSOR_GAMMA_END. Each line in the block processes the first two words as
+    [precursor, gamma]. Populates the lambda field (the precursor production
+    fraction per fission). Symbol \f$ \gamma_j \f$.
+
+  - CHI_DELAYED_BEGIN. Optional. Starts a block that is terminated by a line
+    CHI_DELAYED_END. Each line in the block processes the first word as the
+    group index and the remaining NUM_PRECURSORS words as the the individual
+    precursor's associated delayed spectrum (chi). Populates the chi_d field.
+    Symbol \f$ \chi_{delayed,jg} \f$.
+
+  - TRANSFER_MOMENTS_BEGIN. Optional. Starts a block that is terminated by a line
+    TRANSFER_MOMENTS_END. Each line in the block processes a line only if it
+    starts with the keyword M_GPRIME_G_VAL which needs to be followed by four
+    values [moment,gprime,g,value]. Populates transfer-matrix for moment m,
+    row g, column gprime, with value. Symbol \f$ \sigma_{s\ell,g'{\to}g} \f$.
+- Comments can be between individual blocks but only the TRANSFER_MOMENTS block
+  may have comments between the _BEGIN and _END
+- Comments do not have to start with any specific character since the file
+  format is keyword driven.
+- Any number that is not convertible to its required type (integer, double)
+  will throw an error to that effect.
+- All errors will indicate the current file, line number and nature of the error.
+
+## More Advanced Examples
+\code
+# Bla bla bla
```

Comment:
can we avoid bla bla bla. It was above too... "Place your comments/description in this header portion"

Reply:
Removed.